### DISCUSSION TITLE
Is it possible to insert and update in MongoDB in Redash?

### DISCUSSION THREAD

**Discussion by lscapim** [getredash/redash#6892] [General]
Hi,
Can we do Insert and Updates in MongoDB in Redash or just selects?

**Comment by justinclift** [comment]
Redash is really designed around just doing SELECTS.

So, while you *might* figure out some way to run SQL commands to change things, if that breaks in a future release it's unlikely to be considered a bug to fix.

That being said, what's the scenario you're wanting it for? :smile:

**Reply by lscapim** [reply]
It was just a curiosity, thank you very much!

**Reply by justinclift** [reply]
You're welcome. :smile:
